test/functional/test_framework/test_framework.py

```diff
@@ -83,10 +83,11 @@ def get_default_config_path():
         current_file_path + "/../../../build/test/config.ini",
         current_file_path + "/../../config.ini",
         current_file_path + "/../../../build/x86_64-pc-linux-gnu/test/config.ini",
-        current_file_path + "/../../../build/x86_64-apple-darwin/test/config.ini",
-        current_file_path + "/../../../build/x86_64-w64-mingw32/test/config.ini",
-        current_file_path + "/../../../build/aarch64-linux-gnu/test/config.ini",
         current_file_path + "/../../../build/arm-linux-gnueabihf/test/config.ini",
+        current_file_path + "/../../../build/aarch64-linux-gnu/test/config.ini",
```

Comment:
Let's prioritize the major dev environments first though.. so default selection doesn't pick an unexpected one when you have multiple builds available.
- x64 Linux
- arm64 darwin
- x64 darwin.. 

then the rest..

Reply:
Resolved in  commit 41743bd. Removed old config paths and reordered to prioritize dev builds.